db/schema.rb

```diff
@@ -622,18 +622,38 @@
   add_index "secure_rooms_card_readers", ["tablet_token"], name: "index_secure_rooms_card_readers_on_tablet_token", unique: true, using: :btree
 
   create_table "secure_rooms_events", force: :cascade do |t|
-    t.integer  "card_reader_id",  limit: 4
-    t.integer  "user_id",         limit: 4
+    t.integer  "card_reader_id",  limit: 4,   null: false
+    t.integer  "user_id",         limit: 4,   null: false
```

Comment:
I think we're going to want user_id to be nullable. It's probably fine like this for now, and we can change it back later if we need to.